language-plutus-core/src/Language/PlutusCore/TypeCheck.hs

```diff
@@ -87,7 +87,7 @@ checkKind config ann ty = runTypeCheckM config . checkKindM ann ty
 inferType
     :: (AsTypeError e ann, MonadError e m, MonadQuote m)
     => TypeCheckConfig -> Term TyName Name ann -> m (NormalizedType TyName ())
-inferType config = rename >=> runTypeCheckM config . inferTypeM
+inferType config term = rename term >>= runTypeCheckM config . inferTypeM
```

Comment:
imo this is less clear.

Reply:
This was an accidental change, I've already fixed it.